frontend/app/routes/public/renew/$id/adult-child/confirm-marital-status.tsx

```diff
@@ -0,0 +1,171 @@
+
+  return json({ id: state.id, csrfToken, meta, defaultState: state.hasMaritalStatusChanged, editMode: state.editMode });
+}
+
+export async function action({ context: { appContainer, session }, params, request }: ActionFunctionArgs) {
+  const log = getLogger('renew/adult-child/confirm-marital-status');
+  const state = loadRenewAdultChildState({ params, request, session });
+
+  const t = await getFixedT(request, handle.i18nNamespaces);
+
+  const confirmMaritalStatusSchema = z.object({
+    hasMaritalStatusChanged: z.boolean({
+      errorMap: () => ({ message: t('renew-adult-child:confirm-marital-status.error-message.has-marital-status-changed-required') }),
+    }),
+  });
+
+  const formData = await request.formData();
+  const expectedCsrfToken = String(session.get('csrfToken'));
+  const submittedCsrfToken = String(formData.get('_csrf'));
+
+  if (expectedCsrfToken !== submittedCsrfToken) {
+    log.warn('Invalid CSRF token detected; expected: [%s], submitted: [%s]', expectedCsrfToken, submittedCsrfToken);
+    throw new Response('Invalid CSRF token', { status: 400 });
+  }
+
+  const data = { hasMaritalStatusChanged: formData.get('hasMaritalStatusChanged') ? formData.get('hasMaritalStatusChanged') === 'yes' : undefined };
+  const parsedDataResult = confirmMaritalStatusSchema.safeParse(data);
+
+  if (!parsedDataResult.success) {
+    return json({
+      errors: transformFlattenedError(parsedDataResult.error.flatten()),
+    });
+  }
+
+  saveRenewState({
+    params,
+    session,
+    state: {
+      hasMaritalStatusChanged: parsedDataResult.data.hasMaritalStatusChanged,
+    },
+  });
+
+  if (state.editMode) {
+    return redirect(getPathById('public/renew/$id/adult-child/review-adult-information', params));
+  }
+
+  if (parsedDataResult.data.hasMaritalStatusChanged) {
+    return redirect(getPathById('public/renew/$id/adult-child/marital-status', params));
+  }
+
+  return redirect(getPathById('public/renew/$id/adult-child/confirm-phone', params));
+}
+
+export default function RenewAdultChildConfirmMaritalStatus() {
+  const { t } = useTranslation(handle.i18nNamespaces);
+  const { csrfToken, defaultState, editMode } = useLoaderData<typeof loader>();
+  const params = useParams();
+  const fetcher = useFetcher<typeof action>();
+  const isSubmitting = fetcher.state !== 'idle';
+  const errors = fetcher.data?.errors;
+  const errorSummary = useErrorSummary(errors, {
+    hasMaritalStatusChanged: 'input-radio-has-marital-status-changed-option-0',
+  });
+
+  return (
+    <>
+      <div className="my-6 sm:my-8">
+        <Progress value={22} size="lg" label={t('renew:progress.label')} />
+      </div>
+      <div className="max-w-prose">
+        <p className="mb-4 italic">{t('renew:required-label')}</p>
+        <errorSummary.ErrorSummary />
+        <fetcher.Form method="post" noValidate>
+          <input type="hidden" name="_csrf" value={csrfToken} />
+          <div className="space-y-6">
```

Comment:
```suggestion
          <div className="mb-6">
```

Otherwise the radio buttons are too close to the buttons when editing:
![image](https://github.com/user-attachments/assets/7ee49935-5942-4c9c-8317-6fa0fa4384fd)